paywall/src/middlewares/web3Middleware.js

```diff
@@ -116,10 +117,31 @@ export default function web3Middleware({ getState, dispatch }) {
 
       next(action)
 
+      const {
+        account,
+        router: {
+          location: { pathname, hash },
+        },
+        locks,
+      } = getState()
+      const { lockAddress, transaction } = lockRoute(pathname + hash)
```

Comment:
Is that true on all routes? What happens on `/`?

Reply:
I guess it returns null for both of these?

Reply:
yes